api/src/main/java/io/strimzi/api/kafka/model/connect/ExternalConfiguration.java

```diff
@@ -37,7 +37,7 @@
     private List<ExternalConfigurationVolumeSource> volumes;
     private Map<String, Object> additionalProperties = new HashMap<>(0);
 
-    @Description("Allows to pass data from Secret or ConfigMap to the Kafka Connect pods as environment variables.")
+    @Description("Allows data to pass from a Secret or ConfigMap to the Kafka Connect pods as environment variables.")
```

Comment:
TBH, I do not think this order makes sense to me. Are you sure it is the right way? If yes, I'm fine to go with it. But it sounds weird to me.

Reply:
@tombentley Does the revision sound odd to you?

Reply:
Updated as discussed